src/lib/sensor_calibration/Magnetometer.hpp

```diff
@@ -98,15 +97,21 @@ class Magnetometer
 
 	void Reset();
 
+	void SensorCorrectionsUpdate(bool force = false);
```

Comment:
You'll need to call this periodically for all magnetometers somewhere in VehicleMagnetometer::Run().

Reply:
Done. Thank you for noticing that.